File: javascript/1144.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var valor = parseInt(input);
var [a, b, c] = [1, 1, 1];
var count = 1;
for (let i = 1; i <= valor*2;i++){
    console.log(`${a} ${b} ${c}`);
    if(count%2==0){
        count++;
        a++;
        b+=i;
        c=Math.pow(a,3);
    }else{
        count++;
        b++;
        c++;
    }
}

File: C/1144.c
#include <stdio.h>

int main(){
    int valor, a = 1, b = 1, c = 1, count = 1, i;
    scanf("%d", &valor);
    
    for(i = 1; i <= valor * 2; i++) {
        printf("%d %d %d\n", a, b, c);
        
        if(count % 2 == 0) {
            count++;
            a++;
            b += i;
            c = a * a * a;
        }else {
            count++;
            b++;
            c++;
        }
    }

    return 0;
}